.github/workflows/config/spelling_allowlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.2",
+    "language": "en",
+    "words": [
+        "mlir",
+        "MLIR",
+        "cudaq",
+        "qubit",
+        "Qubit",
+        "qvec",
+        "qref",
+        "canonicalizer",
+        "unsized",
+        "Unsized",
+        "cmake",
+        "CMake",
+        "CMAKE",
```

Comment:
Is there no way to do this case insensitive? (Sorry for the basic question. I've never seen this spelling checker before.)